lib/node/nodes/data-observatory-measure.js

```diff
@@ -29,14 +29,16 @@ var queryTemplate = Node.template([
     '  SELECT ST_SetSRID(ST_Extent(the_geom), 4326) extent, ',
     '         count(*)::INT numgeoms, ',
     '         ARRAY_AGG(DISTINCT ST_GeometryType(the_geom)) AS geom_types, ',
-    '         ARRAY_AGG((the_geom, cartodb_id)::geomval) geomvals ',
+    '         ARRAY_AGG((the_geom, cartodb_id)::geomval) geomvals, ',
+    '         SUM(ST_Area(the_geom)) sumarea ',
```

Comment:
Yes, but, do we expose that normalized number to the user? In that case, and if we expose the normalized value, wouldn't be better to expose a normalized value in some unit that is easier to understand for the user? 🤔